frontend/src/data/config/MetricConfig.ts

```diff
@@ -170,12 +170,12 @@ export const METRIC_CONFIG: Record<string, VariableConfig[]> = {
     {
       variableId: "cases",
       variableDisplayName: "Cases",
-      variableFullDisplayName: "Total COVID-19 Cases",
+      variableFullDisplayName: "COVID-19 Cases", //Total
       metrics: {
         count: {
           metricId: "covid_cases",
-          fullCardTitleName: "Total COVID-19 Cases",
-          shortVegaLabel: "total COVID-19 cases",
+          fullCardTitleName: "COVID-19 Cases", //Total
```

Comment:
Not sure what these comments add

Reply:
those were temporary (where I pulled them out of) until approval from you guys that we removed the correct "totals". will def remove them entirely once we are happy with the results

Reply:
removed